### PR TITLE
add fp32 conversion :sparkles: :sparkles:

### LINKED ISSUE #5: FP32 TensorRT conversion
@shivamswarnkar, Thanks for writing the tensorRT conversion and inference script. However, when I convert the model to fp16 it reduces the accuracy. Is there any configuration so that I convert the model to tensorRT but does not need to change the precision to fp16 and convert the model with fp32. Thank you

_Originally posted by @rashmi-learning in https://github.com/BlueMirrors/Yolov5-TensorRT/issues/4#issuecomment-1027625022_

### PR DESCRIPTION
**Pull request by makaveli10**
added support for fp32 conversion

### CODE REVIEW

File: convert.py [hunk 1]
```diff
@@ -19,7 +19,7 @@ def convert_onnx_to_trt(onnx_weights, image_shape, nc, auto_install=True):
     convert = Yolov5Trt(classes=list(map(str, range(nc))),
                         weight=onnx_weights,
                         backend="tensorrt",
-                        auto_install=auto_install)
```
Comment (shivamswarnkar) [inline convert.py:22]: I think this was deleted by accident?

File: convert.py [hunk 2]
```diff
@@ -43,6 +43,9 @@ def convert_onnx_to_trt(onnx_weights, image_shape, nc, auto_install=True):
     parser.add_argument('--no-auto-install',
                         action='store_true',
                         help="Turn off auto install feature")
+    parser.add_argument('--fp16',
+                        action='store_true',
```
Comment (shivamswarnkar) [inline convert.py:48]: I think default behavior should be ```fp16``` since it's going to be more common. What do you think?

**Review by shivamswarnkar — CHANGES_REQUESTED** [review]

File: convert.py (hunk 2 above)
Comment (ankitnamdeo34) [reply convert.py:48]: [similar suggestion](https://github.com/BlueMirrors/cvu/pull/22#discussion_r799150626) as in pull request https://github.com/BlueMirrors/cvu/pull/22#discussion_r799150626

**Review by shivamswarnkar — APPROVED** [review]
LGTM!
#TODO: Replace ```--float32``` with  ```--dtype``` after next CVU update.

**Review by ankitnamdeo34 — APPROVED** [review]

**Review by aditya-dl — APPROVED** [review]

**Pull request merged**